conf/application.conf

```diff
@@ -108,6 +108,63 @@ microservice {
             searchAgainLinkText = "Search again"
           }
         }
+
```

Comment:
You need to verify how much of this newly introduced configuration actually _changes default values_. It is only necessary to provide configuration where the value is _different_ from the default (see main README). I am concerned that SCRS config is just becoming a copy and paste operation with no thought given to whether the entirety of the pasted config is necessary.